packages/components/src/sandbox/README.md

```diff
@@ -53,4 +53,11 @@ The `<title>` of the iframe document.
 The CSS class name to apply to the `<html>` and `<body>` elements of the iframe.
 
 -   Required: No
--   Default: ""
\ No newline at end of file
+-   Default: ""
+
+### `tabIndex`: `number`
+
+The `tabindex` the iframe should receive.
+
+-   Required: No
+-   Default: 0
```

Comment:
The `tabIndex` prop doesn't actually have a default value in the component, let's remove that line from the README

```suggestion
```

Reply:
Yep, my bad.